playground/android/build.gradle

```diff
@@ -2,7 +2,7 @@
 
 buildscript {
     ext {
-        kotlinVersion = "1.3.72"
+        kotlinVersion = "1.5.31"
```

Comment:
we need to stick to RN version for now https://github.com/facebook/react-native/blob/v0.65.1/packages/react-native-gradle-plugin/build.gradle.kts#L11

Reply:
`1.5` fixes a lot of issues, and `1.5.30` most importantly adds support to Apple Silicon.

Reply:
I have 3 apps running on `1.5.31` without any issues

https://kotlinlang.org/docs/whatsnew15.html
https://kotlinlang.org/docs/whatsnew1530.html

Reply:
lets check that it compiles and tests are ok and then we are gonna figure out to keep 1.5 or use 1.4.3

Reply:
At least it passes all tests on Android 😄
https://github.com/facebook/react-native/pull/32319

Reply:
Added another PR to RN regarding NDK: https://github.com/facebook/react-native/pull/32321
It looks ok too, at least according to tests, hope it will be merged soon, it makes bundle smaller